jbmc/src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -3147,14 +3147,9 @@ bool java_bytecode_convert_methodt::is_method_inherited(
   const irep_idt &classname,
   const irep_idt &methodid) const
 {
-  resolve_inherited_componentt::inherited_componentt inherited_method =
-    get_inherited_component(
-      classname,
-      methodid,
-      symbol_table,
-      class_hierarchy,
-      false);
-  return inherited_method.is_valid();
+  const auto &inherited_method = get_inherited_component(
```

Comment:
Not a reference (0394j4)

Reply:
Initially thought it was C++ spec reference, and was very impressed!